### PR TITLE
Fixing a small typo

### PR DESCRIPTION
**Pull request by virtualandy**
## Which problem is this PR solving?

Eek, I didn't create an issue since this was such a small change. I can definitely do that if needed!

## Description of the changes

Just fixing a typo.

## How was this change tested?

I didn't run locally 😭 since just a typo fix.

## Checklist
- [x] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [x] I have signed all commits
- [ ] I have added unit tests for the new functionality
- [ ] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `yarn lint` and `yarn test`

### CODE REVIEW

**Comment by virtualandy** [comment]
The typo is in versions 1.56 and earlier - if I got this wrong by editing `next-release` let me know and I'm happy to fix it up!

**Review by yurishkuro — COMMENTED** [review]
thanks for catching. Turns out we have an invalid word in the allowed dictionary, and not the only one wrong. We should fix if there first and the linter will flag all occurances.
https://github.com/jaegertracing/documentation/blob/89637b6f45a80e2b4bef55d6d261557a40021345/scripts/cspell/project-words.txt#L26

**Comment by virtualandy** [comment]
@yurishkuro oh, neat, didn't realize that was there (apologies, I should've read all the docs/setup flow first :) ).
I made an update to that word to see how it affects the build/tests. I'll scan it for other typos too.

File: scripts/cspell/project-words.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ datacenter
 datasource
 developercertificate
 dilligence
-distirbuted
+distributed
```
Comment (yurishkuro) [inline scripts/cspell/project-words.txt:26]: I don't believe we need this in the words list. btw dilligence is also misspelled (only needs one l)

**Comment by virtualandy** [comment]
Looks like this was fixed with https://github.com/jaegertracing/documentation/pull/704

**Pull request closed without merging**